From b9cd567cb8ec7c1f1d7c9256ac5e640bb207144a Mon Sep 17 00:00:00 2001
From: Lin Fuzhen <fuzhen.lin@freescale.com>
Date: Wed, 24 Oct 2012 15:05:27 +0800
Subject: [PATCH 357/486] ENGR00230937 powerHAL: use boostpulse interface
 instead of boost

use boostpulse interface instead of boost interface for powerhint,
otherwise cpu will work at hispeed all the time.

Signed-off-by: Lin Fuzhen <fuzhen.lin@freescale.com>
---
 mx6/power/power.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/mx6/power/power.c b/mx6/power/power.c
index 4f18cc3..3c3d04e 100644
--- a/mx6/power/power.c
+++ b/mx6/power/power.c
@@ -20,13 +20,14 @@
 #include <sys/stat.h>
 #include <fcntl.h>
 
-#define LOG_TAG "i.MX PowerHAL"
+#define LOG_TAG "i.MXPowerHAL"
 #include <utils/Log.h>
 
 #include <hardware/hardware.h>
 #include <hardware/power.h>
 
 #define BOOST_PATH      "/sys/devices/system/cpu/cpufreq/interactive/boost"
+#define BOOSTPULSE_PATH "/sys/devices/system/cpu/cpufreq/interactive/boostpulse"
 static int boost_fd = -1;
 static int boost_warned;
 
@@ -88,9 +89,8 @@ static void fsl_power_hint(struct power_module *module, power_hint_t hint,
     case POWER_HINT_VSYNC:
         break;
     case POWER_HINT_INTERACTION:
-	sysfs_write(BOOST_PATH, "1");
+	sysfs_write(BOOSTPULSE_PATH, "1");
 	break;
-
     default:
             break;
     }
-- 
1.8.0

